sdk/src/connections/target/addi9036_sensor.h

```diff
@@ -103,6 +104,7 @@ class Addi9036Sensor : public aditof::DepthSensorInterface,
     std::string m_driverSubPath;
     std::string m_captureDev;
     std::unique_ptr<ImplData> m_implData;
+    long long timestamp;
```

Comment:
You don't really need this. You can obtain the timestamp right in the getFrame() implementation.

Reply:
Yes, indeed. Fixed